docs/site/Advanced-sequence-topics.md

```diff
@@ -0,0 +1,321 @@
+---
+lang: en
+title: 'Advanced Sequence Topics'
+keywords: LoopBack 4.0, LoopBack 4, Node.js, TypeScript, OpenAPI, Sequence
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Advanced-sequence-topics.html
+---
+
+## Customizing Sequence Actions
```

Comment:
Seems to be duplicate content as `Advanced-middleware-based-sequence-topics.md`?